README.md

```diff
@@ -3,6 +3,7 @@
 </p>
 
 # Checkout V2 beta
+The Checkout V2 beta is now available! You have automatically upgraded if you bound to the `master` branch. If you did not mean to upgrade, please bind to `uses: actions/checkout@v1`. Please see the [action versioning docs](https://github.com/actions/toolkit/blob/master/docs/action-versioning.md) for more information.
```

Comment:
Do we need to do this since the examples all reference `@v2-beta`?

This makes me feel like the content is leading with a technical note, rather than substance about the value of checkout v2.

Personal preference, I lean lean toward concise docs. I aim for skimmability (not a word, is now)

Reply:
I viewed it as a short term note just to help those users who auto-upgraded on accident and came to this repo to see what happened, but I get your point. I'll close this pr.